File: src/main/java/com/mymedia/nagasu/dto/ImageCollectionResponse.java
package com.mymedia.nagasu.dto;

import java.util.List;

/**
 * 이미지 컬렉션(폴더) 정보를 반환하는 응답 DTO
 */
public record ImageCollectionResponse(
        String id,
        String name,
        String title,
        String artist,
        List<String> tags,
        String thumbnailUrl,
        int viewCount
) {
}

File: src/main/kotlin/com/mymedia/streamer/dto/ImageCollectionResponse.kt
package com.mymedia.streamer.dto

/**
 * 이미지 컬렉션(폴더) 정보를 반환하는 응답 DTO
 */
data class ImageCollectionResponse(
    val id: String,
    val name: String,
    val title: String,
    val artist: String,
    val tags: List<String>,
    val thumbnailUrl: String
)